llvm/include/llvm-c/Transforms/PassBuilder.h

```diff
@@ -72,6 +72,13 @@ void LLVMPassBuilderOptionsSetVerifyEach(LLVMPassBuilderOptionsRef Options,
 void LLVMPassBuilderOptionsSetDebugLogging(LLVMPassBuilderOptionsRef Options,
                                            LLVMBool DebugLogging);
 
+/**
+ * Specify a custom alias analysis pipeline for the PassBuilder
+ * to be used instead of the default one.
```

Comment:
Maybe add a note here that the string has to live at least as long as PassBuilderOptions, as it is not copied.

Reply:
Maybe it's safer to `strdup` and clean up as part of `LLVMDisposePassBuilderOptions`?